admin/views/partials/_readonly.ejs

```diff
@@ -1,5 +1,12 @@
-<% if (hdfSubmitted) { %>
+<%
+var availabilityData = availabilityData || {};
```

Comment:
This `var` surprised me.  Is there a reason to use `var` over `let`?

Reply:
does the value even change directly?
I think const should be fine?

Reply:
This uses `var` in because `availabilityData` may be undefined.